services/util/SesameWindowed.cpp

```diff
@@ -1,5 +1,5 @@
 #include "services/util/SesameWindowed.hpp"
-#include "infra/stream/BoundedDequeOutputStream.hpp"
+#include "infra/event/EventDispatcherWithWeakPtr.hpp"
```

Comment:
If this is needed at all, then EventDispatcher.hpp is more appropriate than EventDispatcherWithWeakPtr